### PR TITLE
[14.0][IMP] stock_barcodes_advanced: some improvements

### PR DESCRIPTION
**Pull request by ChristianSantamaria**
- Adds qty demand and done in kanban top
- Adds new name_get with "LOT (QTY kg)"

### CODE REVIEW

**Comment by ChristianSantamaria** [comment]
@dalonsod can u check this?
Qty done / demand:
imagen

Order qty and add (qty kg)
imagen

File: stock_barcodes_advanced/models/stock_production_lot.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    _inherit = "stock.production.lot"    
+    
+    def name_get(self):
+        context = self.env.context
+        if context.get('stock_barcodes_advanced', False):
+            result = []
+            for lot in self:
+                name = "%s (%.2f Kg)" % (
+                    lot.name, lot.product_qty
+                )
```
Comment (dalonsod) [inline stock_barcodes_advanced/models/stock_production_lot.py]: Why doesn't use it?
```suggestion
                name = "%s (%.2f %s)" % (
                    lot.name, lot.product_qty, lot.product_uom_id.name
                )
```

File: stock_barcodes_advanced/models/stock_production_lot.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    @api.model
+    def name_search(self, name="", args=None, operator="ilike", limit=100):
+        context = self.env.context
+        if context.get('stock_barcodes_advanced', False):
+            domain = args or []
+            domain += [("name", operator, name)]
+            return self.search(domain).sorted(key=lambda x: x.product_qty, reverse=True).name_get()
+
+        return super().name_search()
```
Comment (dalonsod) [inline stock_barcodes_advanced/models/stock_production_lot.py]: I miss every argument calling this, ignoring them could breack lot search default behavior in other pages:
```suggestion
        return super().name_search(name=name, args=args, operator=operator, limit=limit)
```

File: stock_barcodes_advanced/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -2,3 +2,5 @@ Adds some improvements:
     - Refresh fields after scan
     - Update quantity pending after scan
     - Change barcode button on stock.picking to only confirmed ones
+    - Adds qty demand and done in kanban top
+    - Adds new name_get with "LOT (QTY kg)"
```
Comment (dalonsod) [inline stock_barcodes_advanced/readme/DESCRIPTION.rst]: If suggested change in python file is done, change it:
```suggestion
    - Adds new name_get with "LOT (QTY unit)"
```

File: stock_barcodes_advanced/wizard/stock_barcodes_read_picking_views.xml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+            </xpath>
+            
+        </field>
+    </record>
+
+    <record id="view_stock_barcodes_read_form" model="ir.ui.view">
+        <field name="name">stock.barcodes.read.form</field>
+        <field name="model">wiz.stock.barcodes.read</field>
+        <field name="inherit_id" ref="stock_barcodes.view_stock_barcodes_read_form"/>
+        <field name="arch" type="xml">
```
Comment (dalonsod) [inline stock_barcodes_advanced/wizard/stock_barcodes_read_picking_views.xml:25]: Add a lower priority (e.g. 99), as an attribute replace is performed

**Review by dalonsod — CHANGES_REQUESTED** [review]
See comments, only code review

**Review by dalonsod — APPROVED** [review]
:+1:

**Pull request merged**